src/components/NavbarDev/NavbarStrip.component.tsx

```diff
@@ -16,7 +16,7 @@ const NavbarStrip = () => {
         <span className="mx-2 lg:text-[24px] font-normal leading-[32px]">
           Try OpenMetadata as a managed service for free, from Collate.
         </span>
-        <span className="lg:relative lg:top-[6px]">
+        <span className="lg:relative lg:top-[6px] lg:left-4">
```

Comment:
Why we are adding `lg:left-4`?

Reply:
to add the gap. As we are already using positioning, so to add the gap as per figma, we added this.